Gems/EMotionFX/Code/Source/Integration/Components/ActorComponent.cpp

```diff
@@ -139,19 +139,44 @@ namespace EMotionFX
+                    ->Value("Solid", ActorRenderFlags::Solid)
+                    ->Value("Wireframe", ActorRenderFlags::Wireframe)
+                    ->Value("Lighting", ActorRenderFlags::Lighting)
+                    ->Value("Shadows", ActorRenderFlags::Shadows)
+                    ->Value("FaceNormals", ActorRenderFlags::FaceNormals)
+                    ->Value("VertexNormals", ActorRenderFlags::VertexNormals)
+                    ->Value("Tangents", ActorRenderFlags::Tangents)
+                    ->Value("AABB", ActorRenderFlags::AABB)
+                    ->Value("Skeleton", ActorRenderFlags::Skeleton)
+                    ->Value("LineSkeleton", ActorRenderFlags::LineSkeleton)
+                    ->Value("NodeOrientation", ActorRenderFlags::NodeOrientation)
+                    ->Value("NodeNames", ActorRenderFlags::NodeNames)
+                    ->Value("Grid", ActorRenderFlags::Grid)
+                    ->Value("BackfaceCulling", ActorRenderFlags::BackfaceCulling)
+                    ->Value("ActorBindPose", ActorRenderFlags::ActorBindPose)
+                    ->Value("RagdollColliders", ActorRenderFlags::RagdollColliders)
+                    ->Value("RagdollJointLimits", ActorRenderFlags::RagdollJointLimits)
+                    ->Value("HitDetectionColliders", ActorRenderFlags::HitDetectionColliders)
+                    ->Value("UseGradientBackground", ActorRenderFlags::UseGradientBackground)
+                    ->Value("MotionExtraction", ActorRenderFlags::MotionExtraction)
+                    ->Value("ClothColliders", ActorRenderFlags::ClothColliders)
+                    ->Value("SimulatedObjectColliders", ActorRenderFlags::SimulatedObjectColliders)
+                    ->Value("SimulatedJoints", ActorRenderFlags::SimulatedJoints)
+                    ->Value("EmfxDebug", ActorRenderFlags::EmfxDebug)
+                    ;
+
                 serializeContext->Class<Configuration>()
-                    ->Version(4)
+                    ->Version(5)
                     ->Field("ActorAsset", &Configuration::m_actorAsset)
                     ->Field("MaterialPerLOD", &Configuration::m_materialPerLOD)
-                    ->Field("RenderSkeleton", &Configuration::m_renderSkeleton)
-                    ->Field("RenderCharacter", &Configuration::m_renderCharacter)
-                    ->Field("RenderBounds", &Configuration::m_renderBounds)
                     ->Field("AttachmentType", &Configuration::m_attachmentType)
                     ->Field("AttachmentTarget", &Configuration::m_attachmentTarget)
                     ->Field("SkinningMethod", &Configuration::m_skinningMethod)
                     ->Field("LODLevel", &Configuration::m_lodLevel)
                     ->Field("BoundingBoxConfig", &Configuration::m_bboxConfig)
                     ->Field("ForceJointsUpdateOOV", &Configuration::m_forceUpdateJointsOOV)
+                    ->Field("RenderFlags", &Configuration::m_renderFlags)
```

Comment:
How does this look in the UI? Can we collapse this on default?

Reply:
This is not exposed to UI yet.